earth_enterprise/BUILD.md

```diff
@@ -0,0 +1,57 @@
+If you want to build, install, or run Portable server, see the [Portable Server page](https://github.com/google/earthenterprise/wiki/Portable-Server).
+
+# Building Earth Enterprise Fusion and Server
+
+Building is currently supported on 64-bit versions of Ubuntu 14.04 LTS, Ubuntu 16.04 LTS, RHEL 6, RHEL 7, and CentOS 7.
```

Comment:
Do we want to add CentOS 6 to this list?

Reply:
Update: this will require additional changes to the CentOS/RHEL instructions, including instructions on installing devtoolset-2 on CentOS 6.

Reply:
It's one more command on Cent OS 6. It uses a developer Yum repository.

Reply:
Ok guys - I'm going to give it a go on Centos 6 from scratch - crossing my fingers that I've got enough space for all these VMs!